packages/api-clients/open-api/authorizationApi.yml

```
@@ -430,6 +430,67 @@ paths:
               schema:
                 $ref: "#/components/schemas/Problem"
       description: Add users to a Client
+  "/clients/{clientId}/admin":
+    parameters:
+      - $ref: "#/components/parameters/CorrelationIdHeader"
+      - name: clientId
+        in: path
+        description: ID of Client the users belong to
+        required: true
+        schema:
+          type: string
+          format: uuid
+    post:
+      tags:
+        - client
+      summary: Add Admin to a Client
+      operationId: addAdminToClient
+      requestBody:
+        required: true
+        description: AdminId to be added to the Client
+        content:
+          application/json:
+            schema:
+              type: object
+              additionalProperties: false
+              required:
+                - adminId
+              properties:
+                adminId:
+                  description: UserId to be added as admin
+                  type: string
+                  format: uuid
+      responses:
+        "200":
+          description: Admin added
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Client"
+        "400":
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+        "403":
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+        "404":
+          description: Missing Required Information
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+        "422":
```

```suggestion
        "409":
```
IMHO `Conflict` is more appropriate in this case

---

Right!